101-custom-rp-with-function/src/obj/Debug/netstandard2.0/extensions.AssemblyInfo.cs

```diff
@@ -0,0 +1,23 @@
+//------------------------------------------------------------------------------

```

Comment:
Did you mean to include these in the changes?

Reply:
No, it was Visual Studio Code to include it. If it doesn't go now I will remove the PR. I can't waste all this time. It was working fine, now it's broken because Visual Studio code included changes and I didn't realize it. One more chance than I remove the PR.